src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/DefaultRulesSpec.groovy

```diff
@@ -51,15 +51,15 @@ class DefaultRulesSpec extends HealthCheckSpecification {
         given: "A switch with no rules installed and not connected to the controller"
         def sw = topology.activeSwitches.first()
         northbound.deleteSwitchRules(sw.dpId, DeleteRulesAction.DROP_ALL)
-        Wrappers.wait(RULES_DELETION_TIME) { assert northbound.getSwitchRules(sw.dpId).flowEntries.isEmpty() }
+        wait(RULES_DELETION_TIME) { assert northbound.getSwitchRules(sw.dpId).flowEntries.isEmpty() }
```

Comment:
L21 `import org.openkilda.functionaltests.helpers.Wrappers` is unused now,you can remove it.